### PR TITLE
Add Bottom Navigation

### PR DESCRIPTION
**Pull request by jekingohel**

### CODE REVIEW

File: src/navigation/BottomTabNavigator.tsx [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+
+type ScanTabBarButtonProps = BottomTabBarButtonProps;
+const ScanTabBarButton: React.FC<ScanTabBarButtonProps> = ({ children, onPress }) => (
+    <TouchableOpacity style={{ top: -30, justifyContent: 'center', alignContent: 'center' }} onPress={onPress}>
+        <View
+            style={{
+                width: 70,
+                height: 70,
+                borderRadius: 35,
+                backgroundColor: '#000000',
```
Comment (sadiabbasi) [inline src/navigation/BottomTabNavigator.tsx:37]: use color from theme

File: src/navigation/BottomTabNavigator.tsx [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+    Explore: undefined;
+    Scan: undefined;
+    Apps: undefined;
+};
+
+const Tab = createBottomTabNavigator<RouteStackParamList>();
+
+type ScanTabBarButtonProps = BottomTabBarButtonProps;
+const ScanTabBarButton: React.FC<ScanTabBarButtonProps> = ({ children, onPress }) => (
+    <TouchableOpacity style={{ top: -30, justifyContent: 'center', alignContent: 'center' }} onPress={onPress}>
```
Comment (sadiabbasi) [inline src/navigation/BottomTabNavigator.tsx:31]: don't recomend to add negative top?

Comment (jekingohel) [reply src/navigation/BottomTabNavigator.tsx:31]: As per figma design, QR scan button slightly top to the bottom tab. So its required to use negative top value.

File: src/components/CustomDrawer.tsx [hunk 3]
```diff
@@ -12,6 +12,7 @@ import useWalletStore from '../store/useWalletStore';
 // https://callstack.github.io/react-native-paper/docs/guides/icons/
 const icons: Record<keyof DrawerStackParamList, string> = {
     UserHome: 'home',
+    BottomTabs: 'home',
```
Comment (sadiabbasi) [inline src/components/CustomDrawer.tsx:15]: better to change the value to some thing related to bottomTabs.. may be bottomMenuBar or bottomNavigation something like this?

File: src/navigation/BottomTabNavigator.tsx [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+        <Tab.Navigator
+            screenOptions={{
+                tabBarActiveTintColor: theme.colors.primary,
+                tabBarInactiveTintColor: theme.colors.grey,
+                headerShown: false, // or true if you want headers on tabs
+            }}
+        >
+            <Tab.Screen
+                name="UserHome"
+                component={MainScreen}
```
Comment (sadiabbasi) [inline src/navigation/BottomTabNavigator.tsx:60]: for the citizenship tab we will use the citizen component and name not use the userhome

File: src/navigation/Root.tsx [hunk 5]
```diff
@@ -203,7 +204,7 @@ export default function RootNavigation() {
                 <>
                     <NotificationModule />
                     <CommunicationModule />
-                    <Stack.Navigator initialRouteName={'UserHome'} screenOptions={defaultScreenOptions}>
+                    <Stack.Navigator initialRouteName={'BottomTabs'} screenOptions={defaultScreenOptions}>
```
Comment (sadiabbasi) [inline src/navigation/Root.tsx:207]: with this change after login which screen is display by default?

Comment (jekingohel) [reply src/navigation/Root.tsx:207]: After login it will display Citizenship screen.

File: src/navigation/BottomTabNavigator.tsx (hunk 4 above)
Comment (jekingohel) [reply src/navigation/BottomTabNavigator.tsx:60]: Its updated on Citizenship PR changes.

File: src/components/CustomDrawer.tsx (hunk 3 above)
Comment (jekingohel) [reply src/components/CustomDrawer.tsx]: update to BottomNavigation

File: src/navigation/BottomTabNavigator.tsx (hunk 4 above)
Comment (sadiabbasi) [reply src/navigation/BottomTabNavigator.tsx:60]: I still thinks the same so when we merge this PR aand from the bottom menu when click on scan icon it should open the mainScreen not the citizenship page

File: src/navigation/BottomTabNavigator.tsx [hunk 6]
```diff
@@ -0,0 +1,154 @@
...
+                options={{
+                    tabBarLabel: ({ focused }) => (
+                        <Text
+                            style={[styles.tabLabel, { color: !focused ? theme.colors.tabGray : theme.colors.black }]}
+                        >
+                            Citizenship
+                        </Text>
```
Comment (sadiabbasi) [inline src/navigation/BottomTabNavigator.tsx:67]: this label text repeated may be create a function that return this view and take arguments (label, focused)

File: src/navigation/BottomTabNavigator.tsx (hunk 4 above)
Comment (sadiabbasi) [reply src/navigation/BottomTabNavigator.tsx:60]: and the default page open after just this PR merge will be main screen @theblockstalk am i ritght?

**Pull request closed without merging**